### ISSUE TITLE
Show bootstrap dropdown button in cell ?

### ISSUE THREAD

**Issue by rundis** [facebook/fixed-data-table#226]
I guess this comes down to my lack of css-foo, but I can't figure out how to properly incorporate a [bootstrap dropdown](http://getbootstrap.com/components/#btn-dropdowns) button in a cell. The button renders fine, but due to (guessing) css stacking rules the dropdown menu is not shown (or rather it's hidden behind a range absolute positioned wrapping divs).

any pointers or is this a dead end ?

cheers
magnus

**Comment by ADahmani** [comment]
I have the same problem with my component, have you figured out a solution for this problem ?

**Comment by thechrisproject** [comment]
We ran into this problem. One of our designers had solved it in a similar situation by removing all the overflow:hidden styles and using a series of descending z-indexes on the rows. The proved very difficult. We were going to render the dropdown outside the table and then try to position it on the table when clicking the dropdown arrow, but we just opted for using a very standard dropdown instead; it renders just fine over everything else.

**Comment by ADahmani** [comment]
Yeah, that was what I ended up doing when I had that problem, I noticed that a native normal dropdown works well, so that fixed my problem.

**Comment by cesarandreu** [comment]
You can use [react-overlays](https://github.com/react-bootstrap/react-overlays) (or [react-bootstrap](https://github.com/react-bootstrap/react-bootstrap) if you're using bootstrap styles). They provide an [Overlay](http://react-bootstrap.github.io/react-overlays/examples/#overlay) component. It uses Portal to append a div to the document root and then positions the overlay absolutely. The only problem is that when someone scrolls the overlay stays in the same place...

**Comment by Ghbnxby** [comment]
+1.
You can't just use something like [react-select](https://github.com/JedWatson/react-select) (but i need a component with autocomplete). 
It would be great if the issue was fixed.

**Comment by beharguy** [comment]
+1
I also need a solution for the issue.
It would be great if the issue was fixed.

**Comment by conor909** [comment]
I also found this issue in Internet Explorer, where if the dropdown menu overflows passed the bottom of the table the overflow is hidden.  I can see from the element inspector that div wrapping the table has the in-line styles `-ms-overflow-x: hidden` & `-ms-overflow-y: auto`, I think they should both be `visible` or `inherit` so we can add properties to their parents?

**Comment by warpig9** [comment]
+1

**Comment by Ghbnxby** [comment]
Possible workaround:

```javascript
...
componentDidUpdate(){
    if(this.props.isSelectedCell) {
      //fix z-index
      jQuery(this.refs.cell).parent().parent().parent().parent().parent().parent().parent().parent().parent().parent().css('z-index', 20);
    } else {
      jQuery(this.refs.cell).parent().parent().parent().parent().parent().parent().parent().parent().parent().parent().css('z-index', 0);
    }
  }
...

```
It looks awful, but it works (:

**Comment by fobbyal** [comment]
@Ghbnxby react-select works with that?

**Comment by fobbyal** [comment]
I opened a new issue related to this
#379

**Comment by Ghbnxby** [comment]
Yes. I've added commet to [#379](https://github.com/facebook/fixed-data-table/issues/379) with example
